windows_containers/windows-containers-release-notes-4-x.adoc

```diff
@@ -17,6 +17,26 @@ The release notes for Red Hat OpenShift for Windows Containers tracks the develo
 
 You must have a subscription to receive support for the Red Hat WMCO. Deploying Windows container workloads in production clusters is not supported without a subscription. If you do not have a subscription, you can use the community WMCO, a distribution that lacks official support. Request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
 
+[id="wmco-4-0-1"]
+== Release notes for Red Hat Windows Machine Config Operator 4.0.1
+
+Issued: 2021-12-09
+
+The WMCO 4.0.1 is now available with bug fixes. The components of the WMCO were released in link:https://access.redhat.com/errata/RHBA-2021:4757[RHBA-2021:4757].
+
+[id="wmco-4-0-1-bug-fixes"]
+=== Bug fixes
+
+* Previously, the `windows-exporter` metrics endpoint object contained a reference to a deleted machine. This incorrect reference caused the WMCO to ignore deleted events for machines with invalid IP addresses. This bug fix removes the validation of the machine object from the event filtering, allowing the `windows-exporter` metrics endpoint object to correctly update when the machine is still in the `Deleting` phase. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2008992[**BZ#2008992**])
+
+* Previously, deleting the node associated with a Windows `Machine` object threw a reconciliation error upon restart of the Operator. This bug fix opts not to react or reconcile when the node referenced by a Windows machine in the `Running` state is not found within the cluster, preventing any error loop and standardizing functionality with Linux machine objects. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2009474[**BZ#2009474**])
```

Comment:
```suggestion
* Previously, deleting the node associated with a Windows `Machine` object displayed a reconciliation error upon restart of the Operator. This bug fix opts not to react or reconcile when the node referenced by a Windows machine in the `Running` state is not found within the cluster, preventing any error loop and standardizing functionality with Linux machine objects. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2009474[**BZ#2009474**])
```

Threw? I went with displayed for translation purposes.

Reply:
This was intended to follow the general programming jargon for when an error is "thrown." ISG states to only use that wording if directly referring to a programming language. Seems this is a bit of a tweener, but I'll change it to be safe. :+1:

Reply:
@codyhoag Ah, that makes sense. Hm, maybe (in the future), we could ask someone from the translation team if using "thrown" would cause any issues with translation. Just for my own curiosity. Thank you for explaining :)